test/e2e/specs/specs-playwright/wp-editor__tracks-events.ts

```diff
@@ -0,0 +1,61 @@
+import {
+	DataHelper,
+	LoginFlow,
+	NewPostFlow,
+	GutenbergEditorPage,
+	setupHooks,
+} from '@automattic/calypso-e2e';
+import { Page, Frame } from 'playwright';
+
+describe( DataHelper.createSuiteTitle( `Tracks Events for Post Editor` ), function () {
+	let page: Page;
+	const mainUser = 'gutenbergSimpleSiteUser';
+
+	setupHooks( ( args ) => {
+		page = args.page;
+	} );
+
+	it( 'Log in', async function () {
+		const loginFlow = new LoginFlow( page, mainUser );
+		await loginFlow.logIn();
+	} );
+
+	it( 'Start new post', async function () {
+		const newPostFlow = new NewPostFlow( page );
+		await newPostFlow.newPostFromNavbar();
+	} );
+
+	it( 'Enter post title', async function () {
+		const gutenbergEditorPage = new GutenbergEditorPage( page );
+		await gutenbergEditorPage.enterTitle( DataHelper.randomPhrase() );
+	} );
+
+	it( 'Tracks "wpcom_block_editor_post_publish_add_new_click" event', async function () {
+		const gutenbergEditorPage = new GutenbergEditorPage( page );
+		const frame = await gutenbergEditorPage.getEditorFrame();
```

Comment:
```suggestion
		// Get the frame before publishing because we won't be able to access it once navigation starts
		const frame = await gutenbergEditorPage.getEditorFrame();
```